stable/fairwinds-insights/README.md

```diff
@@ -135,9 +135,11 @@ See [insights.docs.fairwinds.com](https://insights.docs.fairwinds.com/technical-
 | postgresql.replication.enabled | bool | `false` | Replicate Postgres data |
 | postgresql.resources | object | `{"limits":{"cpu":1,"memory":"1Gi"},"requests":{"cpu":"75m","memory":"256Mi"}}` | Resources section for Postgres |
 | encryption.aes.cypherKey | string | `nil` |  |
-| timescale.replicaCount | int | `1` |  |
+| timescale.replicaCount | int | `2` |  |
 | timescale.clusterName | string | `"timescale"` |  |
 | timescale.ephemeral | bool | `true` | Use the ephemeral Timescale chart by default |
+| timescale.pdb.enabled | bool | `true` | Use pdb enabled by default |
+| timescale.pdb.minReplicas | int | `2` | Min timescale pdb replicas |
```

Comment:
This should be `1`--we want to tell the PDB, "allow one of these pods to die, but not both of them"

Reply:
done